src/end.sol

```diff
@@ -43,9 +43,9 @@ contract VatLike {
 }
 contract CatLike {
     function ilks(bytes32) external returns (
-        address flip,  // Liquidator
-        uint256 chop,  // Liquidation Penalty   [ray]
-        uint256 lump   // Liquidation Quantity  [rad]
```

Comment:
Did you mean to remove the comments around these three?

Reply:
Yes